salvador/fetchImage.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  fetchImage.swift
+//  salvador
+//
+//  Created by Geury Roustand on 06.02.23.
+//
+
+import SwiftUI
+
+enum ImageError: Error {
+    case badURL
+}
+
+
+class createImage {
```

Comment:
structs and classes should start with a capital letter (PascalCase). Suggestion: CreateImage